var/spack/repos/builtin/packages/jali/package.py

```diff
@@ -44,15 +44,15 @@ class Jali(CMakePackage):
 
     def cmake_args(self):
         options = []
-        if '+with_mstk' in self.spec:
+        if '+mstk' in self.spec:
             options.append('-DENABLE_MSTK_Mesh=ON')
         else:
             options.append('-DENABLE_MSTK_Mesh=OFF')
```

Comment:
Writing this block as
```python
options.append(self.define_from_variant('ENABLE_MSTK_Mesh', 'mstk'))
```
checks the variant spelling so this won't won't happen again.

Reply:
Thanks. Updated.